app-guides/minio.html.markerb

```diff
@@ -112,15 +117,15 @@ fly deploy
 
 MinIO has a web interface. It's served on the port specified by `--console-address` in the Dockerfile, which we set to `9001`. Fly restricts access to non-standard ports, so to access this panel we need to connect to our private WireGuard network.
```

Comment:
```suggestion
MinIO has a web interface. It's served on the port specified by `--console-address` in the Dockerfile, which we set to `9001`. We can access this panel over our private WireGuard network.
```

I don't remember why that sentence was in there. If we wanted to, we could define a service and make the console public.